src/Microsoft.Health.Fhir.Core/Features/Persistence/ResourceWrapper.cs

```diff
@@ -75,10 +75,10 @@ protected ResourceWrapper()
         }
 
         [JsonProperty(KnownResourceWrapperProperties.LastModified)]
-        public DateTimeOffset LastModified { get; protected set; }
+        public DateTimeOffset LastModified { get; internal set; }
 
         [JsonProperty(KnownResourceWrapperProperties.RawResource)]
-        public RawResource RawResource { get; set; }
+        public RawResource RawResource { get; internal set; }
```

Comment:
>internal set [](http://example.com/codeflow?start=46&length=12)

I needed to update RawResource and LastModified and this class was already not immutable.